programs/stylua.nix

```diff
@@ -185,7 +185,7 @@ in
   config = lib.mkIf cfg.enable {
     settings.formatter.stylua = {
       command = cfg.package;
-      options = lib.mkIf (settingsFile != null) [
+      options = lib.mkIf (settings != { }) [
```

Comment:
I tested this updated version and it breaks the module.

Running `./examples.sh` confirms this. I'm not sure if I am missing something here.

Reply:
I took another look, and your code is good enough.